cobra_commander-ruby/lib/cobra_commander/ruby/version.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module CobraCommander
+  module Ruby
+    VERSION = "0.1.0"
```

Comment:
Any reason for `0.1.0` versus `1.0.0`?

Reply:
@garettarrowood no reason, I'll bump it up.